solver/exporter.go

```diff
@@ -117,6 +117,13 @@ func (e *exporter) ExportTo(ctx context.Context, t CacheExporterTarget, opt Cach
 			return nil, err
 		}
 
+		if e.edge != nil {
+			op, ok := e.edge.op.(*sharedOp)
```

Comment:
looking the source it seems `activeOp` interface is redundant and could be removed, removing the need for this cast.